roles/openshift_openstack/templates/heat_stack.yaml.j2

```diff
@@ -223,6 +223,16 @@ resources:
       # TODO(shadower): Make this configurable?
       lb_algorithm: ROUND_ROBIN
       loadbalancer: { get_resource: api_lb }
+
+  api_lb_health_monitor:
+    type: OS::{{ openshift_openstack_lbaasv2_provider }}::HealthMonitor
+    properties:
+      delay: 5
+      max_retries: 5
+      pool: { get_resource: api_lb_pool }
+      timeout: 2
+      type: HTTPS
```

Comment:
I advise against using HTTPS type of the health monitor - it works differently depending on Octavia version. Which is weird giving that this violates API contract, but what can we do? It's probably best to stick with TCP monitor here. See [cluster-network-operator code](https://github.com/openshift/cluster-network-operator/blob/master/pkg/platform/openstack/loadbalancer.go#L284-L292) for more reference.

Reply:
+1, better to use TCP ones